geode-core/src/main/java/org/apache/geode/management/internal/AgentUtil.java

```diff
@@ -66,11 +68,36 @@ public String findWarLocation(String warFilePrefix) {
     if (possiblePath != null) {
       return possiblePath;
     }
+    // if $GEODE_HOME is not set or we are not able to find it in all the possible locations under
```

Comment:
I just realized that this new code isn't logging the found location anymore. Please can you do that.

Reply:
Done